centreon/src/Centreon/Infrastructure/Contact/ContactRepositoryRDB.php

```diff
@@ -402,8 +402,43 @@ private function createContact(array $contact): Contact
             if (!empty($contact['topology_url_opt'])) {
                 $page->setUrlOptions($contact['topology_url_opt']);
             }
+        } else {
+            if ($contact['contact_template_id'] !== null) {
+                $request = $this->translateDbName(
+                    "SELECT contact.* FROM `:db`.contact WHERE contact_id = :contact_template_id"
```

Comment:
```suggestion
                    "SELECT default_page FROM `:db`.contact WHERE contact_id = :contact_template_id"
```
As you will only use the default_page parameter you could just gather this data instead of all the  column of contact